src/components/views/dialogs/TextInputDialog.js

```diff
@@ -51,6 +52,8 @@ export default class TextInputDialog extends React.Component {
 
         this.state = {
             value: this.props.value,
+            loading: false,
+            buttonValue: this.props.button,
```

Comment:
do we need this or can it be derived based on loading? e.g `this.state.loading ? _t("Loading...") : this.props.button` in render

Reply:
yeah it's not necessary